float64[] next_state
float64 reward
float64 target_cost
float64 unilateral
float64 friction
float64 singularity
float64 joint_range
float64 joint_torques
float64 error_vel_liftoff
float64 error_pos_liftoff
float64 unfeasible_vertical_velocity
float64 no_touchdown
float64 total_cost
---
float64 ack